.github/workflows/apt-packages.yaml

```
@@ -22,7 +22,7 @@ jobs:
       matrix:
         # Debian images:  9 (stretch), 10 (buster), 11 (bullseye)
         # Ubuntu images:  18.04 LTS (bionic), 20.04 LTS (focal), 21.10 (impish), 22.04 (jammy)
```

`21.10 (impish)` could also be removed from this comment

```suggestion
        # Ubuntu images:  18.04 LTS (bionic), 20.04 LTS (focal), 22.04 (jammy)
```